scripts/caravan/impl/actions.lua

```diff
@@ -506,6 +506,33 @@ function P.outpost_item_count(caravan_data, schedule, action)
     end
 end
 
+function P.outpost_fluid_count(caravan_data, schedule, action)
+    local outpost = action.entity
+    if not outpost or not outpost.valid or outpost.name ~= "outpost-fluid" then return false end
```

Comment:
for solids, `get_outpost_inventory` is called here, which is a bit of a misnomer as it works on many other things:

https://github.com/pyanodon/pyalienlife/blob/7f9c532e575132dabb45f8db5432d7d95885d270/scripts/caravan/impl/actions.lua#L7-L23

is there a reason we don't have parity?